pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -364,8 +381,9 @@ type Dependencies struct {
 type SparkPodSpec struct {
 	// Cores is the number of CPU cores to request for the pod.
 	// Optional.
-	// +kubebuilder:validation:Minimum=1
-	Cores *int32 `json:"cores,omitempty"`
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:ExclusiveMinimum=true
```

Comment:
Bizarrely, just setting `Minimum=1` didn't work. With a value of 1, it failed validation with `should be greater than 1` (seems like a bug?)